src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -573,7 +573,8 @@ void createAnalyzerRulesMissingDocumentationFile()
                         helpLinkUri = $"<{helpLinkUri}>";
                     }
 
-                    var line = $"{ruleId} | {helpLinkUri} | {descriptor.Title.ToString(CultureInfo.InvariantCulture)} |";
+                    var escapedTitle = descriptor.Title.ToString(CultureInfo.InvariantCulture).Replace("<T>", "\\<T>");
```

Comment:
Do we want to wait for an issue to appear or try to be a little more generic here? We could potentially do a regex replace of something like `<T[a-zA-Z]*?>`.

Reply:
@Evangelink I wanted to keep it simple, if issues appeared later, either your regex or the regex from markdownlint repository should work.

Reply:
Why not escape all `<` characters?

```suggestion
                    var escapedTitle = descriptor.Title.ToString(CultureInfo.InvariantCulture).Replace("<", "\\<");
```

Reply:
@sharwell Sounds reasonable in the context for rule title.